etcdmain/grpc_proxy.go

```diff
@@ -112,9 +112,9 @@ func newGRPCProxyStartCommand() *cobra.Command {
 	cmd.Flags().StringVar(&grpcProxyDataDir, "data-dir", "default.proxy", "Data directory for persistent data")
 
 	// client TLS for connecting to server
-	cmd.Flags().StringVar(&grpcProxyCert, "cert", "", "identify secure connections with etcd servers using this TLS certificate file")
-	cmd.Flags().StringVar(&grpcProxyKey, "key", "", "identify secure connections with etcd servers using this TLS key file")
-	cmd.Flags().StringVar(&grpcProxyCA, "cacert", "", "verify certificates of TLS-enabled secure etcd servers using this CA bundle")
+	cmd.Flags().StringVar(&grpcProxyCert, "client-cert-file", "", "identify secure connections with etcd servers using this TLS certificate file")
```

Comment:
this cert is used to connect to the etcd backend, right?

client here means the grpc-proxy as a client. i guess it is still confusing. can you check how other reverse proxies name this?